### ISSUE TITLE
JobActivity was removed from neovim with recent job api refactoring

### ISSUE THREAD

**Issue by RyanMcG** [benekastah/neomake#74]
I get the following error now. See neovim/neovim#2247

```
Error detected while processing path/to/neomake/plugin/neomake.vim:
line   31:
E216: No such group or event: JobActivity neomake* call neomake#MakeHandler()
```

**Comment by languitar** [comment]
Same here

**Comment by Neki** [comment]
Should be fixed in #75. Please let me know if there is any further issue. Thanks for reporting!

**Comment by languitar** [comment]
Work well now.
